src/components/TasksList/TasksList.js

```diff
@@ -8,7 +9,7 @@ export const TasksList = () => {
     <div>
       <ul>
         {tasksList.map((task) => (
-          <Task value={task.text} key={task.id} />
+          <Task value={task.text} key={task.id} item={task} />
```

Comment:
`value` duplicates `item`. I would keep only `item`.

```jsx
<Task key={task.id} item={task} />

// in this case your component will look like this:
export const Task = ({ item }) => {
  // ...
};
```

The best practice also is to pass only the required props, e.g.

```jsx
<Task key={task.id} id={task.id} text={task.text} isCompleted={task.isCompleted} />

// or you can pass the whole item, but you should be careful with this.
// It is super handy and super clean, but you may pass accidentally not needed props
<Task key={task.id} {...task} />

// in this case your component will look way more natural
export const Task = ({ id, text, isCompleted }) => {  // props are primitives, which is considered as a best practice
  // ...
};
```